WcaOnRails/config/locales/en.yml

```diff
@@ -288,6 +288,7 @@ en:
         contact: "Contact"
         information: "Information"
         use_wca_registration: "I would like to use the WCA website for registration"
+        is_using_wca_live_for_scoretaking: "I will be using WCA Live for scoretaking"
```

Comment:
```suggestion
```
This locale is under `activerecord`! I think It's supposed to be under `simple_form`.

Reply:
Hi @notrichardpeng, thanks a lot for finding it. This was the issue. Actually I had to put the key in both the places. Under `simple_form`, I kept an empty string as there is no hint needed for that field.

Reply:
ohh ok, I see